README.md

```diff
@@ -13,3 +22,58 @@ Easily create and manage Kubernetes applications using Dekorate and the Halkyon
 - Go to https://circleci.com/gh/halkyonio/hal/tree/master and select the build number you are interested in (presumably, one 
 that succeeded! 😁)
 - Select the `Artifacts` tab and navigate the hierarchy to find the artifact you are interested in.
+
+## Deploying a component using `hal`
+After installing `hal`, the following steps allows to create and deploy a project to a cluster.
+**Note**: this assumes that you are connected to a Halkyon-enabled OpenShift/Kubernetes cluster.
+
+###1 Scaffold the Spring Boot applications 
+
+ - Create a development folder on your laptop
+`mkdir haldemo && cd haldemo`
+
+Create a new project using the REST HTTP `rest` template proposed by the scaffolding tool:
+
+```
+hal component spring-boot \
+    -i hello-world \
+    -g me.example \
+    -p me.example.demo \
+    -s 2.1.6.RELEASE \
+    -t rest \
+    -v 1.0.0-SNAPSHOT \
+    --supported=false \
+    hello-world
+```
+
+###2 Deploy the Component
+
+A component represents a micro-service, i.e. part of an application to be deployed. The Component custom resource provides a simpler to fathom abstraction over what's actually required at the Kubernetes level to deploy and optionally expose the micro-service outside of the cluster. In fact, when a component is deployed to a [Halkyon](https://github.com/halkyonio)-enabled cluster, the [Halkyon operator](https://github.com/halkyonio/operator) will create these OpenShift/Kubernetes resources.
```

Comment:
If you don't enumerate the k8s resources created, then please remove the test at the end of this paragrapha -> "will create these OpenShift/Kubernetes resources."